tests/nn/data_parallel/test_fsdp_optimizer_utils.py

```diff
@@ -36,11 +36,12 @@ def assert_equal(a, b):
 
 class TestOptimizerUtils(DistributedTest):
     @parameterized.expand(
-        [[functools.partial(SGD, momentum=0.9), True], [SGD, False], [Adam, False], [Adadelta, True]],
+        [[functools.partial(SGD, momentum=0.9), True], [SGD, False], [Adam, False], [Adadelta, True], [Adam, True]],
         name_func=rename_test,
     )
     def test_consolidate_optimizer(self, optim_fn, transformer):
         config = {"mixed_precision": True, "flatten_parameters": True}
+        config["compute_dtype"] = torch.float32
```

Comment:
I'm guessing this and the autocast change where required to get tests to pass?

Reply:
yes, I followed the logic of [`__test_identical_outputs`](https://github.com/facebookresearch/fairscale/blob/master/tests/nn/data_parallel/test_fsdp.py#L81)